src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5,6 +5,9 @@
 #define REFERENCE_STATE
 #endif
 
+#nullable enable
+#pragma warning disable 8600
```

Comment:
How did things look with W warnings?

Reply:
There were 43 W warnings in the file.

---
In reply to: [241228286](https://github.com/dotnet/roslyn/pull/31719#discussion_r241228286) [](ancestors = 241228286)